NetworthManager.js

```diff
@@ -0,0 +1,207 @@
+const { calculateNetworth, calculateItemNetworth } = require('./helper/calculateNetworth');
+const { parseItems, postParseItems } = require('./helper/parseItems');
+const { NetworthError, PricesError, ItemsError } = require('./helper/errors');
+const NetworthTypes = require('./NetworthTypes');
+const axios = require('axios');
+const { setItems } = require('./constants/itemsMap');
+
+class NetworthManager {
```

Comment:
So, is the intended use of `NetworthManager` to be a singleton (only 1 instance ever)? If so, maybe there is a better way to enforce this behavior? That way users dont find themselves making a bunch of these classes and wasting memory.

I guess I'm still not sure what the intended purpose of the `NetworthManager` class is. Maybe once I see the README it will help.

Reply:
Yea it is intended for users to only create one. The use case is that there will only be one instance of this. This is good for things like updating items and such.
I added these things because I feel like the getNetworth call got a bit too long with a lot of parameters and options so that we can at least have a bit less cluttered function call.
I do understand that this might not be very nice for some things and I see that especially for things like the net worth type. If you want to have both normal and non cosmetic you need to change the net worth type in the manager before making another call.
If you have any other ideas how we can improve stuff or even a whole other method we can revamp this lmk

Reply:
Yeah, its tricky because we want it to be easier for developers to use while also not being cumbersome or daunting (like with too many arguments) to use

Reply:
The idea of creating a `NetworthManager` class and passing it around all of our files and commands seems challenging as well.